Content.Shared/Hands/EntitySystems/SharedHandsSystem.cs

```diff
@@ -77,6 +77,35 @@ public virtual void RemoveHand(EntityUid uid, string handName, HandsComponent? h
         Dirty(handsComp);
     }
 
+    /// <summary>
+    /// Gets rid of all the entity's hands. Also gets rid of HandsComponent.
+    /// </summary>
+    /// <param name="uid"></param>
+    /// <param name="handsComp"></param>
+
+    public void RemoveHands(EntityUid uid, HandsComponent? handsComp = null)
+    {
+        if (!Resolve(uid, ref handsComp))
+            return;
+
+        RemoveHands(uid, EnumerateHands(uid), handsComp);
+        RemComp(uid, handsComp);
+    }
+
+    private void RemoveHands(EntityUid uid, IEnumerable<Hand> hands, HandsComponent handsComp)
+    {
+        if (!hands.Any())
+            return;
+
+        var hand = hands.First();
+        SetActiveHand(uid, hand, handsComp);
+        DoDrop(uid, hand, handsComp: handsComp);
```

Comment:
```suggestion
```

this is apparently handled by the RemoveHand call
```cs
public virtual void RemoveHand(EntityUid uid, string handName, HandsComponent? handsComp = null)
{
    if (!Resolve(uid, ref handsComp, false))
        return;

    if (!handsComp.Hands.Remove(handName, out var hand))
        return;

    TryDrop(uid, hand, null, false, true, handsComp);
    hand.Container?.Shutdown();
    handsComp.SortedHands.Remove(hand.Name);

    if (handsComp.ActiveHand == hand)
        TrySetActiveHand(uid, handsComp.SortedHands.FirstOrDefault(), handsComp);

    RaiseLocalEvent(uid, new HandCountChangedEvent(uid), false);
    Dirty(handsComp);
}
```

Reply:
Alright give me like half an hour and I'll do it